kged/src/components/titlebar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { setGameName } from '../actions/global'
+import '../styles/titlebar.scss'
+
+export class TitleBar extends React.Component{
+    constructor(props){
+        super(props)
+        this.onChangeHandler = this.onChangeHandler.bind(this)
+    }
+
+    onChangeHandler = event => {
+        event.preventDefault()
+        this.props.setName(event.target.value)
+    }
+
+    render(){
+        return(
```

Comment:
Small inconsistency with existing code style.

```suggestion
    render() {
        return (
```